db/migrate/20170717140915_add_numeric_columns_to_container_quota_items.rb

```diff
@@ -0,0 +1,44 @@
+class AddNumericColumnsToContainerQuotaItems < ActiveRecord::Migration[5.0]
+  class ContainerQuotaItem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    add_column :container_quota_items, :quota_desired_float,  :float
+    add_column :container_quota_items, :quota_desired_int,    :bigint
```

Comment:
> * These come from quotas ultimately set by kubernetes users, who can express same quantity as 2048000m cpu cores or 2Ki cores. The first would parse as 2048.0 float, the second as 2048 int, but I don't see why keep the distinction to any code that will deal with these columns...

It doesn't matter how to the user expresses them, of course Kubernetes is not keeping the user's string, it's transforming it in a `Quantity` which will be then returned with the most appropriate formatting.

> * AFAIK milli is the only sub-integer unit in this use case, so if we really wanted no rounding we could store everything in 1000x scaled fixed point. Ugly but better than 2 columns IMHO.

OK for using floats. I think that the rounding takes place somewhere around ~`2**50` (please verify this).
It would be nice to have this check as a test, e.g. store a large integer (something like `2**50`) and check that reading the resulting float is still `==` to the original integer.